### DISCUSSION TITLE
How to export a functioning Skeleton.hkx with ragdoll functions?

### DISCUSSION THREAD

**Discussion by CoffeeIsDead** [BadDogSkyrim/PyNifly#202] [Q&A] [answered]
Hello, I need to generate a skeleton.hkx for a mod I'm making. I was able to export one but it lacks ragdoll functions and causes T-posing in game. Is there a guide or resource I can look at find how to do this. I've tried google but all the tuts I can find are for 3dsMax/Maya and Havok tools for them.

**Comment by CoffeeIsDead** [comment]
Running FNIS Doesn't solve the T-posing

**Comment by BadDogSkyrim** [comment, marked as answer]
This isn't something I know how to do. I'm not even sure how ragdoll functions are done. I'm guessing if you translate the hkx to xml, you can then pick out the ragdoll elements and add them to your skeleton. But I really don't know.
